src/devices/sound/segapcm.cpp

```diff
@@ -35,10 +35,20 @@ void segapcm_device::device_start()
 {
 	m_stream = stream_alloc(0, 2, clock() / 128);
 	
-	std::fill(std::begin(m_ram), std::end(m_ram), 0xff);
+	m_ram = make_unique_clear<uint8_t[]>(0x800);
```

Comment:
Why are you using `make_unique_clear` here when you plan to fill it with 0xff at reset time anyway?  It's wasted work to zero-fill something when you plan to completely overwrite it anyway.